pkg/apis/autoscaling/v1/fleetautoscaler.go

```diff
@@ -151,7 +151,19 @@ type BufferPolicy struct {
 // WebhookPolicy controls the desired behavior of the webhook policy.
 // It contains the description of the webhook autoscaler service
 // used to form url which is accessible inside the cluster
-type WebhookPolicy admregv1.WebhookClientConfig
+type WebhookPolicy struct {
+	admregv1.WebhookClientConfig `json:",inline"`
+
+	// Fallback defines how the autoscaler should behave in the event the webhook fails.
+	// +optional
+	Fallback *WebhookFallback `json:"fallback,omitempty"`
```

Comment:
I'll ask the question just to be sure -- would we ever want a fallback for other policies? Or just webhook? (i.e. should this be further up).

I'm fairly sure the answer is "no", but wanted to triple check just in case.

Reply:
IMO no. The other policies are purely internal and cannot really fail. This is unique to Webhooks.